.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+dist: trusty
+
+language: python
+python: 3.5
+
+install:
+  # Install code sniffer.
+    pip install pycodestyle;
+    pip install --upgrade pycodestyle;
+    source ~/.bash_profile;
+    source ~/.bashrc;
+
+script:
+  # Run code sniffer on the scraper directory.
+    pycodestyle --show-source --max-line-length=119 --show-pep8 .
+    
```

Comment:
There is 2 `enters` at the EOF (End Of File), There is no need for such a whitespace, one is enough.